packages/matchbox/src/components/Button/styles.js

```diff
@@ -0,0 +1,136 @@
+import { tokens } from '@sparkpost/design-tokens';
+
+export const base = (props) => `
+    display: inline-block;
+    border-radius: ${tokens.borderRadius_100};
+    user-select: none;
+    font-weight: ${tokens.fontWeight_medium};
+    text-decoration: none;
+    white-space: nowrap;
+    vertical-align: middle;
+    transition-property: background, color, border, outline;
+    transition-duration: .15s;
+    border: 1px solid transparent;
+  `;
+
+export const size = (props) => {
+  switch (props.visualSize) {
+    case 'large':
+      return `
+        line-height: calc(3.5rem - 2px);
```

Comment:
Is this to handle text wrapping or vertical centering? If the latter, the component should just be styled using `display: inline-flex` and `align-items: center` instead. I could be missing something though.

Reply:
The `-2px` is there to ensure the buttons are a certain height (`3.5rems`) with the `1px` borders. Let me see if I can do this with inline-flex